Basic-Scripts/balanced_paranthesis.py

```diff
@@ -1,15 +1,28 @@
-open_bracket_list = ["[", "{", "("]
-close_bracket_list = ["]", "}", ")"]
+openBracketList = ["[", "{", "("]
+closeBracketList = ["]", "}", ")"]
 
 
-def checkParanthsis(data):
+def checkParanthsis(data: str) -> str:

```

Comment:
Follow [Function Name Guidelines](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names) for the function name and check spellings.